android/gradle.properties

```diff
@@ -12,10 +12,8 @@ org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryErro
 # org.gradle.parallel=true
 #Tue Dec 19 15:08:27 EST 2023
 KlaviyoReactNativeSdk_klaviyoAndroidSdkVersion=3.0.3
-KlaviyoReactNativeSdk_compileSdkVersion=31
-KlaviyoReactNativeSdk_kotlinVersion=1.8.0
+KlaviyoReactNativeSdk_kotlinVersion=1.7.0
 KlaviyoReactNativeSdk_minSdkVersion=23
-KlaviyoReactNativeSdk_ndkversion=21.4.7075529
 KlaviyoReactNativeSdk_targetSdkVersion=31
-android.enableJetifier=true
-android.useAndroidX=true
```

Comment:
No support for RN libs on AndroidX?
Should we keep these in but set them explicitly to false?

Reply:
[See above](https://github.com/klaviyo/klaviyo-react-native-sdk/pull/194/files/ac39379d5a026c168733ed08eb68fc8bbb7ad5c2#r1937781180)

> Looking back in git, it appears I added them pretty early on, but I'm not clear why. They weren't part of the original template, nor are they in the latest one. And I'm not sure why we'd need them, since the SDK's android files do not make use of androidx at all.
The example app does have useAndroidx=true which makes sense because it has androidx dependency for the activities and notifications and such.
So I'm thinking this is safe to remove.

Reply:
Ha, I should have refreshed before I commented! Thanks for clarifying